Dockerfile

```
@@ -1,6 +1,7 @@
 FROM java:7-jre-alpine
 
-RUN apk --no-cache add curl unzip
+RUN apk update && \
+    apk --no-cache add curl unzip tar
```

I prefer listing dependencies alphabetically with one dependency per line to improve diffs.

```suggestion
    apk --no-cache add \
      curl \
      tar \
      unzip
```